stdlib/SuiteSparse/src/cholmod.jl

```diff
@@ -1450,51 +1450,69 @@ it should be a permutation of `1:size(A,1)` giving the ordering to use
 (instead of CHOLMOD's default AMD ordering).
 
 # Examples
-```jldoctest
-julia> A = [4. 12. -16.; 12. 37. -43.; -16. -43. 98.]
-3×3 Array{Float64,2}:
-   4.0   12.0  -16.0
-  12.0   37.0  -43.0
- -16.0  -43.0   98.0
 
-julia> p = [2, 3, 1]
-3-element Array{Int64,1}:
- 2
- 3
- 1
+In the following example, the fill-reducing permutation used is `[3, 2, 1]`.
+If `perm` is set to `1:3` to enforce no permutation, the number of nonzero
+elements in the factor is 6.
+```jldoctest
+julia> A = [2 1 1; 1 2 0; 1 0 2]
```

Comment:
Maybe `A = sparse([2 1 1; 1 2 0; 1 0 2])` and remove the two `sparse(A)` calls below?

Reply:
I did this to have the nice printing of `A` below to help visualize the sparsity of `A`

Reply:
Eh, just leave it like it is then.